src/commands/lambda/prompt.ts

```diff
@@ -24,6 +17,10 @@ import {
 } from './constants'
 import {sentenceMatchesRegEx} from './functions/commons'
 
+// @ts-ignore
```

Comment:
I know using the ignore is kinda hacky but I this is the way I was able to get it to work.
Without the ignore I get
```
Could not find a declaration file for module 'inquirer-checkbox-plus-prompt'. '/Users/andrew.rodriguez/go/src/github.com/DataDog/datadog-ci/node_modules/inquirer-checkbox-plus-prompt/index.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/inquirer-checkbox-plus-prompt` if it exists or add a new declaration (.d.ts) file containing `declare module 'inquirer-checkbox-plus-prompt';`ts(7016)
```

Where would I put this .d.ts file?

Reply:
You can use `require` to avoid the `ts-ignore` (it may require to disable `eslint` though) but if the type is simple, you should indeed add your `d.ts` (next to that file for instance).